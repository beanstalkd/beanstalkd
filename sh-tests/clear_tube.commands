use tubea
put  1 0 0 4
data
put  1 0 0 4
data
put  1 0 0 4
data
put  1 0 0 4
data
put  1 0 0 4
data
put  1 0 0 4
data
watch tubea
clear-tube tubea
reserve-with-timeout 0
put  1 0 0 2
ok
reserve-with-timeout 0
quit
