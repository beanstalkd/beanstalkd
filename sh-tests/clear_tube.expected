USING tubea
INSERTED 1
INSERTED 2
INSERTED 3
INSERTED 4
INSERTED 5
INSERTED 6
WATCHING 2
CLEARED
TIMED_OUT
INSERTED 7
RESERVED 7 2
ok
